### PR TITLE
[DQL2-12] 5 Star rating system - Automated testing considerations

### PR DESCRIPTION
**Pull request by MarkCalvert**
@duttonw @ThrawnCA Here is the PR for [DQL2-12] 5 Star rating system - Automated testing considerations
- Setup ckan job workers for QA BDD tests
- Added CKAN config for archiver and qa
- Added data usability rating BDD tests and resource files

### CODE REVIEW

File: .ahoy.yml [hunk 1]
```diff
@@ -121,8 +119,40 @@ commands:
   test-bdd:
     usage: Run BDD tests.
     cmd: |
+      ahoy start-ckan-job-workers &
```
Comment (ThrawnCA) [inline .ahoy.yml:122]: This probably doesn't need to be backgrounded when the workers are individually backgrounded.

File: .ahoy.yml [hunk 2]
```diff
@@ -121,8 +119,40 @@ commands:
...
+      ahoy title 'Starting CKAN background job workers'
+      ahoy start-ckan-job-worker-priority &
+      ahoy start-ckan-job-worker-bulk &
+
+  start-ckan-job-worker-priority:
+    usage: Starts CKAN background job worker
+    cmd: |
+      ahoy title 'Starting CKAN background job worker priority'
+      ahoy cli "cd /usr/lib/ckan/default/src/ckan && \
+                paster jobs clear priority --config=/app/ckan/default/production.ini && \
```
Comment (ThrawnCA) [inline .ahoy.yml:140]: Why clear the queue?

Comment (duttonw) [reply .ahoy.yml:140]: most likely to reset testing.

Comment (MarkCalvert) [reply .ahoy.yml:140]: @ThrawnCA It is really just there to be safe for local dev testing when sometimes the worker jobs were still running and did not stop successfully giving me peculiar results. 
In circleci this is not an issue as the containers are created before and destroyed after the tests complete but on local dev/testing this is not always the case.
I think its safer and cleaner with this approach for local BDD dev/testing and no harm with leaving in.

File: .ahoy.yml (hunk 1 above)
Comment (MarkCalvert) [reply .ahoy.yml]: @ThrawnCA Yes you are correct and I have removed it.

**Pull request merged**